downstream/modules/platform/proc-edge-manager-request-cert.adoc

```diff
@@ -0,0 +1,52 @@
+[id="edge-manager-request-cert"]
+
+= Optional: Requesting an enrollment certificate for early binding
+
+If you want to include an agent configuration in the image, complete the following steps:
+
+.Procedure
+
+. Authenticate with the {RedHatEdge} service by using the `flightctl` CLI. Run the following command:
```

Comment:
Suggest changing to:

Authenticate with the {RedHatEdge} service by using the `flightctl` CLI by running the following command: